src/uct/tcp/tcp.h

```diff
@@ -452,6 +453,25 @@ typedef struct uct_tcp_iface_config {
 } uct_tcp_iface_config_t;
 
 
+/**
+ * @brief TCP MD descriptor
+ */
```

Comment:
```suggestion
/**
 * TCP memory domain
 */
```
to align with TCP iface